utils/utils.go

```diff
@@ -48,6 +48,9 @@ const (
 
 	windowsOsType = "windows"
 	homeDirPrefix = "~/"
+
+	// DefaultChannelAddress is the default local network address that user application listen on.
+	DefaultChannelAddress = "127.0.0.1"
```

Comment:
Not that this is likely to change, but can this be used from the dapr/dapr package? Its already defined there

Reply:
If possible I do not want to create another dependency in runtime. Can we move certain constants to kit and let both runtime and CLI depend on that instead?

Reply:
What's the issue with adding a package from `dapr/dapr`? In Go, as long as you don't have a package inside `internal`, it's perfectly fine to use packages. This ensures we have consistency and not minimize chances of non-synced changes. If anything, I would advocate for the CLI to rely on `dapr/dapr` packages as much as it possibly can.

Reply:
Importing `github.com/dapr/pkg/runtime` into utils.go for using the constant defined there started creating issue wrt duplicate protobuf registration https://github.com/dapr/cli/actions/runs/5129434834/jobs/9227056977 associated commit https://github.com/dapr/cli/pull/1283/commits/0381466d446d6eca9eeda9bade0c23095046c168

for now hardcoding that constant to 127.0.0.1 directly in CLI itself.